### PR TITLE
Add Nobø Ecohub integration

### PR DESCRIPTION
**Pull request by oyvindwe**
## Proposed change

Documentation for new climate integration: Nobø Ecohub

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/50913
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2603
- This PR fixes or closes issue: -

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by oyvindwe** [comment]
As far as I can tell, this PR is correctly rebased.

File: source/_integrations/nobo_hub.markdown
Comment (frenck) [inline source/_integrations/nobo_hub.markdown]:
```suggestion
The possible operation modes are as follows:

```

File: source/_integrations/nobo_hub.markdown
Comment (frenck) [inline source/_integrations/nobo_hub.markdown]:
```suggestion
- "Auto" - In this mode, the zone is in the normal setting and preset shows which state the zone is in right now
  (according to calendar setup).
```

File: source/_integrations/nobo_hub.markdown
Comment (frenck) [inline source/_integrations/nobo_hub.markdown]:
```suggestion
- "Heat" - In this mode the zone in the override setting and in the state selected by preset ("Away", "Eco"
  or "Comfort").
```

File: source/_integrations/nobo_hub.markdown
Comment (frenck) [inline source/_integrations/nobo_hub.markdown]:
```suggestion
- "Off" - In this optional mode the zone is in the normal setting and will configure it to use a week profile that
  turns the heater(s) in the zone completely off (more info below).
```

File: source/_integrations/nobo_hub.markdown
Comment (frenck) [inline source/_integrations/nobo_hub.markdown]:
```suggestion

- Changing preset to "Away", "Eco", or "Comfort" will automatically change operation to "Heat".
- Changing preset to none will automatically change operation to "Auto" and update preset.
- Changing operation to "Auto" will automatically update preset.
- Changing operation to "Heat" will set preset to "Comfort".
- Changing operation to "Off" will change operation to "Auto" and change the week profile to the predefined
  "completely off" profile.
```

File: source/_integrations/nobo_hub.markdown
Comment (frenck) [inline source/_integrations/nobo_hub.markdown]:
```suggestion

```

File: source/_integrations/nobo_hub.markdown
Comment (frenck) [inline source/_integrations/nobo_hub.markdown:22]:
```suggestion
IP address of the hub.

{% include integration/config_flow.md %}

## Operation modes
```

File: source/_integrations/nobo_hub.markdown
Comment (frenck) [inline source/_integrations/nobo_hub.markdown]:
```suggestion
Nobø heaters can normally not be set to override "off". This is not a limitation in the integration, but a
safety mechanism in the Nobø system (maybe they don't want you to accidentally turn off all your heaters and get
frozen pipes). However, it is possible to create a week profile that makes the heaters stay off all the time. And then
you can configure the system to switch a zone to this week's profile to be able to turn the heater(s) off.
```

File: source/_integrations/nobo_hub.markdown
Comment (frenck) [inline source/_integrations/nobo_hub.markdown]:
```suggestion
profile for each of your zones, you can use this module to turn off (and on) your heaters. The week profiles
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Please note: You need to open a PR at the brands repository to move the brand images from a custom integration to a core integration.

**Comment by oyvindwe** [comment]
Thanks for the review! 

> Please note: You need to open a PR at the brands repository to move the brand images from a custom integration to a core integration.

Thank you - I didn't know that, as the logo appeared automatically in my dev environment. Should I move or copy the logo?

File: source/_integrations/nobo_hub.markdown
Comment (frenck) [inline source/_integrations/nobo_hub.markdown]:
```suggestion
  "completely off" profile.

## There is no override in Nobø Ecohub for "completely off", but there is a workaround
```

**Comment by frenck** [comment]
The brand images should be moved.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by frenck — DISMISSED** [review]

**Comment by oyvindwe** [comment]
Thank you for the review!

> ⚠️ The `ha_release` might need an update before merging this one in.

I guess we don't know that until the parent PR is merged?

**Comment by frenck** [comment]
> I guess we don't know that until the parent PR is merged?

Correct.

File: source/_integrations/nobo_hub.markdown
Comment (MartinHjelmare) [inline source/_integrations/nobo_hub.markdown]:
```suggestion
ha_release: 2022.10
```

File: source/_integrations/nobo_hub.markdown
Comment (MartinHjelmare) [inline source/_integrations/nobo_hub.markdown:9]: We didn't add echoromeo as codeowner in the integration I believe.

File: source/_integrations/nobo_hub.markdown
Comment (MartinHjelmare) [inline source/_integrations/nobo_hub.markdown:47]:
```suggestion
Nobø heaters do not support preset "Off". This is not a limitation in the integration, but a safety mechanism in the
```

**Comment by MartinHjelmare** [comment]
Try rebasing on latest `next` branch to make the build pass.

**Comment by oyvindwe** [comment]
> Try rebasing on latest `next` branch to make the build pass.

Done.

**Review by MartinHjelmare — DISMISSED** [review]
Thanks!

File: source/_integrations/nobo_hub.markdown
Comment (frenck) [inline source/_integrations/nobo_hub.markdown]:
```suggestion
ha_release: 2021.10
```

**Review by frenck — APPROVED** [review]
Thanks, @oyvindwe 👍

**Pull request merged**
